app-shells/zsh/zsh-5.5.1.recipe

```diff
@@ -92,6 +96,7 @@ BUILD()
 		--with-tcsetpgrp \
 		--enable-zsh-secure-free
 
+	sed --in-place '/HAVE_WAIT3/d' config.h
```

Comment:
I thought you decided to remove this? Is there any reason why this is coming back?